modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/metadata/MetadataExtensionFunctionalTestCase.java

```diff
@@ -151,6 +151,11 @@ public void setup() throws Exception {
     personType = getMetadata(PERSON_METADATA_KEY.getId());
   }
 
+  @Override
```

Comment:
I went through every TestCase and flagged those that could use this configuration without side-effects. For Metadata, every TestCase can use this.